odxtools/cli/_print_utils.py

```diff
@@ -96,7 +98,7 @@ def print_service_parameters(service: DiagService,
         print_fn(f"  Positive Response '{resp.short_name}':")
         print_fn(f"   Parameters:\n")
         table = extract_parameter_tabulation_data(list(resp.parameters))
-        print_fn(Padding(table, pad=(0, 0, 0, 4)))
+        print_fn(RichPadding(table, pad=(0, 0, 0, 4)))
```

Comment:
print_fn or rich_print?

Reply:
print_fn is passed as an argument to the print_service_parameters function. maybe this should be changed to hardcoded `rich_print`. done: https://github.com/mercedes-benz/odxtools/pull/362/commits/433e234c0c2313eaaee3ffc0ea74e93978cdf8b7, thanks for the catch.